frontend/src/index.jade

```diff
@@ -42,7 +42,7 @@ html
           .tab-content
             .tab-panes
               #tab-authorship.tab-pane(v-if="isTabAuthorship")
-                .toolbar(v-if="tabInfo.tabAuthorship.totalCommits > 0")
+                .toolbar(v-if="hasCommits(tabInfo.tabAuthorship)")
```

Comment:
i refactored the toolbar to put it inside vauthorship. you might wanna take a look at the implementation, can just pull into here if you are using it.